src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,122 @@
+package duke.ui;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The user interface for the Chat Bot.
+ *
+ * @author Armando Jovan Kusuma
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________";
+    private static final String MESSAGE = LINE + "\n"
+            + " Hello! I'm ChatBot\n"
+            + " What can I do for you?\n"
+            + LINE + "\n";
+
+    /**
+     * Greets the user at the start
+     */
+    public void greet() {
+        System.out.println(MESSAGE);
+    }
+
+    /**
+     * Prints a message if a task is added to the list
+     *
+     * @param input the task added to the list of tasks
+     * @param count the number of tasks in the list
+     */
+    public void taskPrint(Task input, int count) {
+        System.out.println(LINE + "\n"
+                + "Got it. I've added this task" + "\n"
+                + input + "\n" + "Now you have " + count
+                + " tasks in this list." + "\n" + LINE);
+    }
+
+    /**
+     * Prints a message if a task is deleted from the list
+     *
+     * @param input the task deleted from the list of tasks
+     * @param count the number of tasks in the list
+     */
+    public void deletePrint(Task input, int count) {
+        System.out.println(LINE + "\n"
+                + "Noted. I've removed this task: " + "\n"
+                + input + "\n" + "Now you have " + count
+                + " tasks in this list." + "\n" + LINE);
+    }
+
+    /**
+     * Prints a message if a task is marked as done.
+     *
+     * @param task the task to be marked as done
+     */
+    public void markDonePrint(Task task) {
+        System.out.println(LINE);
```

Comment:
The `System.out.println(LINE)` before and after the output seems to be repeated in several places. Perhaps you could abstract it into a separate method to reduce code duplication.